Bottom: 688b39e938726ffd1598cac97bafb7840cc57b6e
Top:    290971ee8274335209baf855f7366a32637e5294
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   2011-03-26 16:13:10 +0000

Additional defines for a 3D joystick for (older) linux versions

Older versions of linux do not have these defines in <linux/input.h>

#define REL_RX  0x03
#define REL_RY  0x04
#define REL_RZ  0x05



---

diff --git a/src/hid/gtk/snavi.c b/src/hid/gtk/snavi.c
index bd50813..7d3f961 100644
--- a/src/hid/gtk/snavi.c
+++ b/src/hid/gtk/snavi.c
@@ -21,6 +21,19 @@
 
 #include <linux/types.h>
 #include <linux/input.h>
+/*
+Additional defines for a 3D joystick for (older) linux versions which do not
+have these defines in <linux/input.h>
+*/
+#ifndef REL_RX
+#define REL_RX                  0x03
+#endif
+#ifndef REL_RY
+#define REL_RY                  0x04
+#endif
+#ifndef REL_RZ
+#define REL_RZ                  0x05
+#endif
 
 #include <glib.h>
